iam_profile_faker/factory.py

```diff
@@ -28,9 +29,18 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def create_random_hierarchy_iter():
+    """Generate hierarchy iterator with a random pattern pattern"""
+    def gen():
+        for i in itertools.count():
+            yield (i + 1, random.randint(0, i))
```

Comment:
Although this generator works (we have a root and then all next ``i`` have a random pick in ``(0, i]`` ) have you considered that the data might not be realistic?

Reply:
It should be good enough to start with. I already have a dump of the current real orgchart in this pattern format and I would like to add it here as an option.